frontend/app/src/store/defi/index.ts

```diff
@@ -358,7 +356,7 @@ export const useDefiStore = defineStore('defi', () => {
           summary[protocol].tokenInfo?.tokenName !== entry.baseBalance.tokenName
         ) {
           const tokenInfo: Writeable<TokenInfo> = summary[protocol].tokenInfo!;
-          tokenInfo.tokenName = `${i18n.t('defi_overview.multiple_assets')}`;
+          tokenInfo.tokenName = `${t('defi_overview.multiple_assets')}`;
```

Comment:
other way to call .toString() ?  😛

Reply:
Yes I did that at some point, this is a leftover